posthog/batch_exports/http.py

```diff
@@ -355,7 +355,7 @@ class BatchExportViewSet(TeamAndOrgViewSetMixin, viewsets.ModelViewSet):
     queryset = BatchExport.objects.exclude(deleted=True).order_by("-created_at").prefetch_related("destination").all()
     serializer_class = BatchExportSerializer
     scope_object_read_actions: list[str] = ["retrieve", "list"]
-    scope_object_write_actions: list[str] = ["destroy", "create", "backfill", "pause", "unpause"]
+    scope_object_write_actions: list[str] = ["destroy", "create", "backfill", "pause", "unpause", "patch"]
```

Comment:
tbh this exactly the reason why I don't want to use this way of doing scopes (I commented in the previous PR). Instead we should remove these alltogether and add the specific scopes to the backfill and pause actions instead 
```
@action(methods=["POST"], detail=True, required_scopes=["batch_exports:write"])
```

Can I ask for that change instead 😬 ?

Reply:
Made the change